test/creator.cpp

```diff
@@ -198,6 +198,7 @@ TEST(ZimCreator, createZim)
   item = std::make_shared<TestItem>("foo2", "AFoo", "Foo2Content");
   creator.addItem(item);
   creator.addMetadata("Title", "This is a title");
+  creator.addIllustration(48, 48, "PNGBinaryContent");
```

Comment:
This test nicely demonstrates that `addIllustration()` is not fool-proof. Sure enough, there will be users running against this imperfection, we may have extra issues in the bug-tracker and ZIM archives with invalid illustration content.